Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -262,15 +307,32 @@ private IConfidentialClientApplication BuildConfidentialClientApplication()
                 request.Scheme,
                 request.Host,
                 request.PathBase,
-                azureAdOptions.CallbackPath ?? string.Empty);
+                azureAdOptions.CallbackPath.Value ?? string.Empty);
+
+            string authority = string.Empty;
+            IConfidentialClientApplication app = null;
+
+            if (azureAdOptions.IsB2C)
+            {
+                authority = $"{applicationOptions.Instance.TrimEnd('/')}/tfp/{azureAdOptions.Domain}/{azureAdOptions.DefaultPolicy}";
```

Comment:
>/tfp/ [](start = 72, length = 5)

do we need this still? `/tfp/` is only used for the login.microsoftonline.com instances.